docs/advisors/checks/configuration-check-sync-binlog.md

```diff
@@ -12,4 +12,10 @@ and [Percona blog posts on the topic](https://www.percona.com/blog/2018/05/04/ho
 `SELECT @@global.sync_binlog;`
 
 ## Resolution
-Consider setting the **sync_binlog** variable to **1** with `SET GLOBAL sync_binlog=1`.
+Disable localhost bypass:
+1. Edit mongod.conf and set the following parameter:
+```setParameter:
+           enableLocalhostAuthBypass: false
+```
+
+2. Perform a rolling restart of your mongod nodes to apply the changes.
```

Comment:
This looks like a resolution for another check. Probably it got mixed up with the `mongodb_localhost_bypass` one.